### ISSUE TITLE
Environment variables in env files

### ISSUE THREAD

**Issue by pldin601** [alwex/php-database-migration#14]
Is it possible to use environment variables in env configurations? I populated database credentials into system env and would like to use it in migrations. Thanks!

**Comment by alwex** [comment]
It is not possible right now, but it sounds like a good idea.
The configuration loader use `symfony/config` and `symfony/yaml`, I will see if it can support environment variable out of the box.

Feel free to open a pull request if you want to contribute to the project.

**Comment by pldin601** [comment]
I think will be nice to add a .php configuration files support. Then we may use expressions inside config:

```php
<?php

return [
    'connection' => [
        'host' => $_ENV['DB_HOST'] ?? 'localhost',
        'driver' => 'pgsql',
        'username' => $_ENV['DB_USERNAME'] ?? 'root'
        ....
    ]
];
```

**Comment by pldin601** [comment]
https://github.com/alwex/php-database-migration/pull/15

**Comment by alwex** [comment]
fixed with 3.7.0
